playbooks/roles/backups/files/backup.py

```diff
@@ -74,7 +74,8 @@ def upload_to_gcloud_storage(file_path, bucket):
     """
 
     import boto
-
+    import gcs_oauth2_boto_plugin
```

Comment:
@amirtds Could you give some comment why this import fixes the issue?  I don't see the imported module used anywhere.

Reply:
Thanks @thraxil for answering Bryan's question